indico/modules/events/registration/__init__.py

```diff
@@ -240,3 +241,17 @@ class RegistrationPermission(ManagementPermission):
     friendly_name = _('Registration')
     description = _('Grants management access to the registration form.')
     user_selectable = True
+
+
+@signals.event.updated.connect
```

Comment:
I need to take care of date/time changes and other info like title or logo change... 
Ok, I'll work on that.

Reply:
I don't think we have a signal for logo changes. Probably not really likely to happen once there are tickets. Anyway, feel free to trigger the event.updated signal in case of logo changes

Reply:
OK... let's do like this: forget about the logo changes for the moment... eventually I'll come back with another PR to add the event.updated signal in the future :)

Reply:
This was not resolved... `times_changed` is still not handled...